sysvar/src/lib.rs

```diff
@@ -145,11 +142,24 @@ pub fn is_sysvar_id(id: &Pubkey) -> bool {
     ALL_IDS.iter().any(|key| key == id)
 }
 
+/// Interface for loading a sysvar.
+pub trait SysvarGet: Default + Sized {
```

Comment:
We removed the requirement for the `Default` trait in [pinocchio](https://github.com/anza-xyz/pinocchio/pull/189), so I wonder if we should do the same here as well. Individual sysvars could still implement `Default` when applicable.